commit 9f6c9a8c50bc84ec748fec779ead321ee2b2debc
tree e0e58850265784fc6a4515c7cc637179d430a8d3
parent ceded32f097ba753fb4d339ad5c74f265ea8a03d
author Masakazu Mokuno <mokuno@sm.sony.co.jp> 1184919654 +0900
committer Jeff Garzik <jeff@garzik.org> 1185308919 -0400

    ps3: fix wrong calculation of rx descriptor address
    
    Fixed the bug that calculation of the address of rx descriptor was
    wrong.
    
    Signed-off-by: Masakazu Mokuno <mokuno@sm.sony.co.jp>
    Signed-off-by: Jeff Garzik <jeff@garzik.org>

---
 drivers/net/gelic_net.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/gelic_net.c	2007-08-27 14:01:21.000000000 -0400
+++ b/drivers/net/gelic_net.c	2007-08-27 14:03:58.000000000 -0400
@@ -1109,7 +1109,7 @@ static int gelic_net_open(struct net_dev
 			card->descr, GELIC_NET_TX_DESCRIPTORS))
 		goto alloc_tx_failed;
 	if (gelic_net_init_chain(card, &card->rx_chain,
-				 card->descr + GELIC_NET_RX_DESCRIPTORS,
+				 card->descr + GELIC_NET_TX_DESCRIPTORS,
 				 GELIC_NET_RX_DESCRIPTORS))
 		goto alloc_rx_failed;
 
